### PR TITLE
Metadata - Re-ordering

### PR DESCRIPTION
**Pull request by technicalgrp89**
all general-purpose metadata definitions are in-
tuna/utils/metadata.py
tests/ test case - correction (metadata path)

### CODE REVIEW

**Review by cderb — APPROVED** [review]

File: tuna/utils/metadata.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+""" This file contains mappings relevant to launching MIOpenDriver and interacting with sql tables
```
Comment (alexandraBara) [inline tuna/utils/metadata.py]:
```suggestion
""" This file contains mappings relevant to launching workers and interacting with sql tables
```

File: tuna/worker_interface.py [hunk 2]
```diff
@@ -49,8 +49,10 @@
 from tuna.machine import Machine
 
 from tuna.abort import chk_abort_file
-from tuna.miopen.utils.metadata import TUNA_LOG_DIR
-from tuna.miopen.utils.metadata import NUM_SQL_RETRIES
+from tuna.utils.metadata import TUNA_LOG_DIR
+from tuna.utils.metadata import NUM_SQL_RETRIES
+from tuna.utils.metadata import MAX_JOB_RETRIES
+from tuna.utils.metadata import LOG_TIMEOUT
```
Comment (alexandraBara) [inline tuna/worker_interface.py]: is there a reason to put each import in a separate line?

Comment (alexandraBara) [inline tuna/worker_interface.py]:
```suggestion
from tuna.utils.metadata import TUNA_LOG_DIR, MAX_JOB_RETRIES, NUM_SQL_RETRIES, LOG_TIMEOUT
```

File: tuna/miopen/utils/helper.py [hunk 3]
```diff
@@ -39,7 +39,7 @@
 from tuna.miopen.utils.metadata import MYSQL_LOCK_WAIT_TIMEOUT
 from tuna.miopen.utils.metadata import BN_DEFAULTS
 from tuna.miopen.utils.metadata import FUSION_DEFAULTS, CONV_2D_DEFAULTS, CONV_3D_DEFAULTS
-from tuna.miopen.utils.metadata import NUM_SQL_RETRIES
+from tuna.utils.metadata import NUM_SQL_RETRIES
```
Comment (iq136boy) [inline tuna/miopen/utils/helper.py:42]: Why we keep two metadata.py scripts? Any specific reason to create the new one?

Comment (technicalgrp89) [reply tuna/miopen/utils/helper.py:42]: we have moved the common definitions to tuna.utils.metaday.py.
no two instances

**Review by iq136boy — APPROVED** [review]

**Comment by technicalgrp89** [comment]
@alexandraBara pls recheck the changes.

File: tuna/miopen/db/session.py [hunk 4]
```diff
@@ -38,6 +38,8 @@ class Session(BASE, SessionMixin):
   """Session table to keep track of tunning sesions"""
   #pylint: disable=attribute-defined-outside-init
   #pylint: disable=too-many-instance-attributes
+  #pylint: disable=no-name-in-module
+  #pylint: disable=no-value-for-parameter
```
Comment (msaudulhassan) [inline tuna/miopen/db/session.py]: Do you know why the linter is complaining about these?

Comment (alexandraBara) [reply tuna/miopen/db/session.py]: I am also curious what these 2 are about

Comment (technicalgrp89) [reply tuna/miopen/db/session.py]: I think they were added to solve other errors. Now I see no dependency on these Pylint warning.

**Review by iq136boy — APPROVED** [review]

**Review by cderb — APPROVED** [review]

**Review by msaudulhassan — APPROVED** [review]

**Review by alexandraBara — APPROVED** [review]

**Pull request merged**